Button/Button.svelte

```diff
@@ -0,0 +1,18 @@
+<script>
+    export let text
+    export let outline
+</script>
+
+<style type="text/scss" lang="scss">
+    @import '../styles/variables.scss';
+    button {
+        padding: 8px 12px;
+    }
+    .outline {
+        background: none;
+        border: 1px solid;
+        color: red;
+    }
+</style>
+
+<button class:outline on:click>{text}!</button>
```

Comment:
Please use a `slot` instead of `text` prop.

```html
<button class:outline on:click>
  <slot></slot>
</button>
```

Reply:
Definitely, this is just an illustration of what the proposed feature is look like. The issues filed for the first two components are still subject of debate and approval.